classes/omnect_uboot_embedded_version.bbclass

```diff
@@ -146,8 +150,22 @@ python omnect_uboot_embed_version() {
         bb.fatal("Unable to stat bootloader binary \"%s\"" % (version_file))
     filesize = st.st_size
 
+    # do we need an end tag which will be searched for by uuu tool during
+    # initial flashing in production process?
+    # NOTE:
+    #   this handling is normally found in yocto layer meta-freescale in
+    #   classes file uuu_bootloader_tag.bbclass.
+    #   in order to be independend of thos meta layer this is handled here,
+    #   too, we're processing the binary here anyway.
```

Comment:
do we have cases supporting an imx device without meta-freescale?

Reply:
yes, dehndetect does not currently use/need meta-freescale.